examples/operations/QueueScript.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>Consider:</p><p>When validating the configuration we will get....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>